### ISSUE TITLE
confusion matrix for all classes when run evaluation on test set and validation set 

### ISSUE THREAD

**Issue by sulaiman1988** [meituan/YOLOv6#991]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

how to print confusion for all classes  not just print all 
 Class         Images      Instances          Box(P              R          mAP50      mAP50-95)         Mask(P              R        mAP50    mAP50-95)
all               1623           7628               0.80948     0.8051      0.84518         0.71888            0.81388       0.79858    0.83438    0.65959
class A 
class B
class C

**Comment by Chilicyy** [comment]
Hi @sulaiman1988 , you could try to add `--verbose` when run evaluation.
